eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Compiler/TranslationPass.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * File containing the TranslationPass class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Compiler;
+
+use eZ\Bundle\EzPublishCoreBundle\Translation\Collector;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * This compilation pass loads every ezplatform available translations into symfony translator.
+ */
+class TranslationPass implements CompilerPassInterface
```

Comment:
@bdunogier  I thought about it at the begining but after having a look at the other compiler pass in the core i guess it's more a generic naming about the composant it targets like : 

- AsseticPass
- ChainRoutingPass
- LocalePass
- SecurityPass
....

I didn't named it TranslatorPass cause it deal with translation so.